pypdf/_protocols.py

```diff
@@ -59,6 +59,8 @@ def strict(self) -> bool:
 
 
 class PdfReaderProtocol(PdfCommonDocProtocol, Protocol):
+    strict: bool
```

Comment:
Why hasn't this been required before?

Reply:
Strange... now it seems to not be required...
Will remove it